chain/chain/Cargo.toml

```diff
@@ -73,6 +73,9 @@ test_features = [
     "node-runtime/test_features",
 ]
 shadow_chunk_validation = []
+protocol_feature_spice = [
```

Comment:
Why did we choose to have a feature flag instead of protocol feature for spice?

Reply:
using protocol feature requires to have `protocol_version` available, so we start with cargo feature to reduce the amount of changes and then over time we will move to protocol feature